app/rifcs/rif_cs_generator.rb

```diff
@@ -130,4 +138,12 @@ def build_rif_cs
     end
   end
 
+  private
+
+  def get_title(url)
+    page = open(url,:allow_redirections => :safe)  #this can raise exception
+    @doc = Nokogiri::HTML(page)
+    return @doc.xpath("//title").text
```

Comment:
Catch the exception here and return an empty string - the caller shouldn't have to deal with it.